src/Parallel/GlobalCache.hpp

```diff
@@ -100,8 +87,120 @@ using get_component_if_mocked = tmpl::front<tmpl::type_from<tmpl::conditional_t<
     tmpl::lazy::find<ComponentList,
                      tmpl::type_<get_component_if_mocked_helper<
                          tmpl::pin<ParallelComponent>, tmpl::_1>>>>>>;
+
 }  // namespace GlobalCache_detail
 
+template <typename Metavariables>
+class MutableGlobalCache : public CBase_MutableGlobalCache<Metavariables> {
```

Comment:
I put all the doxygen in GlobalCache, because end users don't see MutableGlobalCache and access everything thru GlobalCache.  But I can add some doxygen in MutableGlobalCache too if that would be less confusing.

Reply:
I think it would be helpful to include a little documentation, but I'd be happy with just a short summary of what it does, and that it is meant to be used only via the GlobalCache functions.